### DISCUSSION TITLE
How to get world position from mouse position?

### DISCUSSION THREAD

**Discussion by DJDQfff** [stride3d/stride#2226] [Q&A] [answered]
The method like `Camera.main.ScreenToWorldPoint(mousepos);` in Unity?

**Comment by Doprez** [comment]
Not built in but we have a community toolkit of extensions like these. [This](https://github.com/stride3d/stride-community-toolkit/blob/53bff1597c8698eeb8176ef7a95add612687c72e/src/Stride.CommunityToolkit/Engine/CameraComponentExtensions.cs#L194) is the code for the screen to world that you are looking for.

**Comment by IXLLEGACYIXL** [comment, marked as answer]
https://github.com/stride3d/stride-community-toolkit/blob/53bff1597c8698eeb8176ef7a95add612687c72e/src/Stride.CommunityToolkit/Engine/CameraComponentExtensions.cs#L103
and this is the method to shoot a ray from the mouse position

https://www.nuget.org/packages/Stride.CommunityToolkit/1.0.0-preview.38
and here is the nuget
